demo/md-styles.html

```diff
@@ -0,0 +1,787 @@
+<link rel="import" href="../polymer/polymer-element.html">
```

Comment:
If this was sourced from higlightjs, we have 2 situations here:
1. Faster - write a simplified version of this style file built from scratch (original work)
2. Slower - I have to get a lawyer involved to resolve the legal attributions for this file (if we can accept it)

I'm happy to do either one, but what path would you like to take @chalu